frontend/src/lib/components/Link.tsx

```diff
@@ -16,15 +16,14 @@ export function Link({ to, preventClick = false, tag = 'a', ...props }: LinkProp
 
         if (!props.target) {
             event.preventDefault()
-            event.stopPropagation()
```

Comment:
Why?

Reply:
The context on this change is that when we have links inside tooltips, popovers, dropdowns (referring to Ant components), the elements do not get dismissed after clicking the link, because the click does not get propagated. I do believe it makes sense to propagate the event normally. The reason for the `Link` component is to use `kea-router` instead of a browser's native navigation, this is achieved with only the `event.preventDefault()`.

In addition, this component creates an `a` tag for most cases and even when doing a `LinkButton` (which uses an underlying `button` component) forms won't get submitted.

The above being said, there might be a case I might be failing to consider.

Reply:
Hmm. I would have expected LinkButton to be breaking - if not I don't see this breaking anything, though it is a change which might sneakily break something.

Reply:
I specifically tested `LinkButton` and it did not trigger a form submission, but I do agree we might be inadvertently introducing a bug somewhere. Though I think it's a good opportunity to make this change and fix later if something actually breaks. The alternative is just defaulting to stopPropagation and adding an extra prop that can stop that behavior (I actually did this at first), but seems like it would introduce painful tech debt.